src/style.css

```diff
@@ -1 +1,105 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Heavy.ttf') format('truetype');
+  font-weight: bold;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir.ttc') format('truetype');
+  font-weight: 400;
+  font-style: normal;
+}
+
+body {
+  font-family: Avenir, Arial, sans-serif;
```

Comment:
Consider adding fallback font families after 'Avenir' to ensure text remains readable in case the custom font fails to load. For example, you might use a generic font family like 'sans-serif' as a fallback.